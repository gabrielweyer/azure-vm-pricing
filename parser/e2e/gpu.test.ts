import assert from "./assertExtensions";

describe('End-to-end tests for instances with GPUs', () => {
  const culture = 'en-us';
  const currency = 'usd';
  const operatingSystem = 'windows';

  test('No reserved instances', (done) => {
    assert(done, culture, currency, operatingSystem, 'H16mr');
  });
});
