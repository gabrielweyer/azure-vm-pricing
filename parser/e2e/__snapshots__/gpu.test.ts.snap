// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`End-to-end tests for instances with GPUs No reserved instances 1`] = `"H16mr,16,224,4.742,2.861,N/A,N/A,N/A,N/A,0.5833,0.2861,3.1175,2.3815,2.501,1.765"`;
